dist/api_data.py

```diff
@@ -1740,6 +1736,31 @@ def __ge__(self, other):
         dropped if open in a cursor. While a hot backup is in progress, checkpoints created
         prior to the start of the backup cannot be dropped''',
         type='list'),
+    Config('flush_tier', '', r'''
+        configure flushing objects to tiered storage after checkpoint''',
+        type='category', subconfig= [
+            Config('enabled', 'false', r'''
+                if true, perform one iteration of object switching and flushing objects to
+                tiered storage''',
+                type='boolean'),
+            Config('force', 'false', r'''
+                if false (the default), flush_tier may be skipped if the underlying object
+                has not been modified since the previous flush_tier. If true, this option forces
+                the flush_tier''',
+                type='boolean'),
+            Config('sync', 'true', r'''
+                wait for all objects to be flushed to the shared storage to the level specified.
+                When false, do not wait for any objects to be written to the tiered storage system
+                but return immediately after generating the objects and work units for an internal
+                thread.  When true,the caller waits until all work queued for this call to be
+                completely processed before returning''',
+                type='boolean'),
+            Config('timeout', '0', r'''
+                maximum amount of time to allow for waiting for previous flushing of objects, in
+                seconds. The actual amount of time spent waiting may exceed the configured value. A
+                value of zero disables the timeout''',
+                type='int'),
```

Comment:
I would not use the word "maximum" if the actual amount of time may exceed it.  I think we should say that we return EBUSY if the timeout is reached.

Reply:
Fixed.